lib/ex_unit/lib/ex_unit/case.ex

```diff
@@ -11,9 +11,12 @@ defmodule ExUnit.Case do
 
   When used, it accepts the following options:
 
-    * :async - configure this specific test case to able to run in parallel
+    * `:async` - configure this specific test case to able to run in parallel
```

Comment:
Yeah, it definitely depends. I tried to convey this "depends" idea by using "It's usually a good idea" because I think it *usually* is 😄, but maybe we can find a better wording? I just care about mentioning that asyncness can interfere with capturing io/logs :)